src/Resources/doc/index.rst

```diff
@@ -0,0 +1,912 @@
+NelmioSecurityBundle
+====================
+
+The NelmioSecurityBundle provides additional security features for your Symfony application.
+
+Installation
+------------
+
+Require the ``nelmio/security-bundle`` package in your composer.json and update
+your dependencies:
+
+.. code-block:: terminal
+
+    $ composer require nelmio/security-bundle
+
+The bundle should be automatically enabled by ``Symfony Flex``_. If you don't use
```

Comment:
```suggestion
The bundle should be automatically enabled by `Symfony Flex`_. If you don't use
```
this is compared with other links (it may be fine, I'm not familiar with `rst`)

Reply:
Good catch! Thanks for the review. This is fixed now.